### ISSUE TITLE
Bug in UI of settings toggle

### ISSUE THREAD

**Issue by mly32** [perfect-things/perfect-home#137]
# Bug

## Problem

The toggles in the settings act like sliders when the expected behavior is that they are toggles.

As they are right now if the setting is "off", the only way to turn the setting "on" is to click on the right side of the sider or to drag the slider. Clicking on the left side (the "off" side) should still toggle the setting to "on". Similarly, when the setting is "on", clicking on the right side should still toggle the setting to "off". Another effect of the current UI is that clicking on the side that does not work causes a glitchy animation.

```html
<input id="showLabels" type="range" min="0" max="100" class="toggle">
```
![gif example](https://s9.gifyu.com/images/Animation7c69524f1194fc1c.gif)
Gif example: [here](https://s9.gifyu.com/images/Animation7c69524f1194fc1c.gif).

## Solutions

Here are some suggestions:

1. Change the underlying input type. Here's a good example: [https://www.w3schools.com/howto/howto_css_switch.asp](https://www.w3schools.com/howto/howto_css_switch.asp).
2. Add some javascript to handle on-click and similar events.

**Comment by tborychowski** [comment]
Hi.

This was intentional.
I know many UI frameworks fucked-up the toggle, but it was never supposed to be a click-to-toggle widget!
It came from mobile uis, where it used to behave exactly like this one - you slide to switch. 
It was based on a physical, real world switch, which you "slide" to the opposite position.

That said, I know that even mobile UIs have now extended the behaviour of this, so it can be used with click-to-toggle :-(
So maybe it's time to give up to bootstraps of this world 😉 

I may add the click-to-toggle but the sliding will stay too, as it's the right UX to do.

(and btw. s3schools is not a good example of anything. It has a bad reputation in the dev community, especially on the expert level, for mistakes and blunders in their docs)

**Comment by mly32** [comment]
Oh, that's cool. I forgot about the mobile side of things. I think you can keep the dragging behavior.

Not sure how useful this is: [https://v5.material.angular.io/components/slide-toggle/examples](https://v5.material.angular.io/components/slide-toggle/examples), but they have the behavior I was describing. Underlying the toggle is a checkbox, but they keep the dragging behavior of the toggle through CSS styling.

Feel free to close this issue.

**Comment by tborychowski** [comment]
Yep, the material example is exactly what I intent to do: it allows the click but also drag!
:-)

**Comment by tborychowski** [comment]
That's fixed now and available in v0.13.5.
Thanks for pushing me :-)
